incubator/jaeger/values.yaml

```diff
@@ -105,10 +105,10 @@ ingress:
   # Used to create an Ingress record.
   # hosts:
   #   - chart-example.local
-  #annotations:
+  # annotations:
```

Comment:
This (and the following) changes aren't really part of this PR, but the linter failed here.

```
Linting the Chart.yaml and values.yaml files at incubator/jaeger                                                             
incubator/jaeger/values.yaml                                                                             
  108:4     error    missing starting space in comment  (comments) 
  111:4     error    missing starting space in comment  (comments)                                       
```